lib/beacon/loader/routes.ex

```diff
@@ -49,6 +49,17 @@ defmodule Beacon.Loader.Routes do
           @endpoint.url() <> beacon_media_path(file_name)
         end
 
+        def beacon_page_url(%{path: path} = page) do
+          prefix = @router.__beacon_scoped_prefix_for_site__(@site)
```

Comment:
Better use `Phoenix.VerifiedRoutes.unverified_path/3` here, for an example see: https://github.com/BeaconCMS/beacon/blob/ba5e3be248d942120c9842aa12c83061f0dce7c9/lib/beacon/web/components/layouts.ex#L33-L34

Same for `beacon_sitemap_url`